installers/unix/env/install.sh

```diff
@@ -79,39 +79,54 @@ check_dep $PYTHON
 python -c "import sys; sys.exit('Anaconda Python isn\'t supported by this package. Please install the official Python from python.org.') if 'conda' in sys.version else ''"
 
 if [ -z $DEST ]; then
-    PROMPT="Type the path where you want to install OpenQuake, followed by [ENTER]. Otherwise leave blank, it will be installed in $HOME/openquake: "
+    PROMPT="Type the path where you want to install OpenQuake, followed by [ENTER]. Otherwise leave blank, it will be installed in ${HOME}/openquake: "
     read -e -p "$PROMPT" DEST
     [ -z "$DEST" ] && DEST=$HOME/openquake
 fi
 FDEST=$(realpath "$DEST")
 
-echo "Creating a new python environment in $FDEST. Please wait."
+echo "Creating a new python environment in ${FDEST}. Please wait."
 /usr/bin/env $PYTHON virtualenv/virtualenv.py $FDEST > /dev/null
-cp -R {README.md,LICENSE,demos,doc} $FDEST
+
 
 [ $MACOS ] && \
-    cat <<EOF >> $FDEST/env.sh
+    cat <<EOF >> ${FDEST}/env.sh
     export LC_ALL=en_US.UTF-8
     export LAN=en_US.UTF-8
 EOF
 
-cat <<EOF >> $FDEST/env.sh
-. $FDEST/bin/activate
+cat <<EOF >> ${FDEST}/env.sh
+. ${FDEST}/bin/activate
 EOF
 
-source $FDEST/env.sh
-echo "Installing the files in $FDEST. Please wait."
+source ${FDEST}/env.sh
+echo "Installing the OpenQuake Engine. Please wait."
 # Update pip first
 /usr/bin/env pip install --disable-pip-version-check -U wheelhouse/pip*.whl > /dev/null
 /usr/bin/env pip install --disable-pip-version-check wheelhouse/*.whl > /dev/null
+mkdir -p $FDEST/share
+cp -R src/{README.md,LICENSE,demos,doc} $FDEST/share
 
+## Tools installation
```

Comment:
`TOOLS` variable logic is not so intuitive, please add a comment explaining if 'n' or 'N'  => false else true

Reply:
Done